modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -617,33 +617,35 @@ const EditorManager: EditorManager = {
    * @method execCommand
    * @param {String} cmd Command to perform for example Bold.
    * @param {Boolean} ui Optional boolean state if a UI should be presented for the command or not.
-   * @param {String} value Optional value parameter like for example an URL to a link.
+   * @param {any} value Optional value parameter like for example an URL to a link.
```

Comment:
```suggestion
   * @param {Object/String/Number} value Optional value parameter like for example an URL to a link.
```
Sorry I missed this previously, but remember we can't use TypeScript types in API docs. Since we can't use that, I think this is probably going to be the closest we can use.